src/main.c

```diff
@@ -804,6 +854,7 @@ int main(int argc, char *argv[])
     showmsg(tlfversion);
     showmsg("");
 
+    total = 0;
```

Comment:
I think we should change the readcall() funtion to made it more explicit. I will look for that myself.